1.22.0

matteobruni opened the pull request:
feat: added noise/path generator in plugins management

matteobruni committed:
build: schema

matteobruni committed:
feat: added noise/path generator in plugins management

ghost commented:
## DeepCode's analysis on [#951937](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/e80d8e17bd7a1ab2038498de18ee17b8fb85add7/matteobruni/tsparticles/95193711a1afb0ac6c848068e2dc504e7212dc45/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) found:

- :information_source: **1** minor issue. :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">'public' is implicit. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/95193711a1afb0ac6c848068e2dc504e7212dc45/core/main/src/Utils/Plugins.ts#L19">Plugins.ts:19</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/95193711a1afb0ac6c848068e2dc504e7212dc45/core/main/src/Utils/Plugins.ts#L23">Plugins.ts:23</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/95193711a1afb0ac6c848068e2dc504e7212dc45/core/main/src/Utils/Plugins.ts#L29">Plugins.ts:29</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/e80d8e17bd7a1ab2038498de18ee17b8fb85add7/matteobruni/tsparticles/95193711a1afb0ac6c848068e2dc504e7212dc45/pr/_/%2Fcore%2Fmain%2Fsrc%2FUtils%2FPlugins.ts/javascript%2Ftslint%2Fmember-access/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/e80d8e17bd7a1ab2038498de18ee17b8fb85add7/matteobruni/tsparticles/95193711a1afb0ac6c848068e2dc504e7212dc45/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
Merge branch 'main' into v1

matteobruni commented:
This will implement also the request made here https://github.com/VincentGarreau/particles.js/issues/463 by @nelsonmic, thanks for the idea

matteobruni commented:
This version is going to make v1 compatible with the `sea anemone` sample/preset made in v2

matteobruni committed:
refactor: renamed noise to path in move options, kept compatibility

matteobruni committed:
build: fixed noise/path options and editor

matteobruni committed:
build: updated old noise usage

matteobruni committed:
chore(release): published new version

 - ng-particles@2.5.0
 - inferno-particles@1.5.0
 - jquery-particles@1.22.0
 - preact-particles@1.22.0
 - react-tsparticles@1.22.0
 - svelte-particles@1.5.0
 - particles.vue@2.5.0
 - particles.vue3@1.5.0
 - tsparticles-editor@1.22.0
 - tsparticles@1.22.0
 - ng-particles-demo@1.7.0
 - editor-demo@1.7.0
 - inferno-particles-demo@1.5.0
 - jquery-particles-demo@1.7.0
 - tsparticles-demo@1.7.0
 - preact-particles-demo@1.7.0
 - react-tsparticles-demo@1.7.0
 - svelte-demo@1.5.0
 - particles.vue-demo@1.7.0
 - particles.vue3-demo@1.5.0
 - tsparticles-preset-60fps@1.7.0
 - tsparticles-preset-background-mask@1.7.0
 - tsparticles-preset-basic@1.7.0
 - tsparticles-preset-big-circles@1.7.0
 - tsparticles-preset-bouncing@1.7.0
 - tsparticles-preset-bubbles@1.7.0
 - tsparticles-preset-fire@1.7.0
 - tsparticles-preset-font-awesome@1.7.0
 - tsparticles-preset-fountain@1.7.0
 - tsparticles-preset-sea-anemone@1.7.0
 - tsparticles-preset-snow@1.7.0
 - tsparticles-preset-stars@1.7.0
 - tsparticles-shape-bubble@1.7.0
 - tsparticles-shape-confetti@1.7.0
 - tsparticles-shape-heart@1.7.0
 - tsparticles-shape-multiline-text@1.7.0
 - tsparticles-shape-rounded-rect@1.7.0
 - tsparticles-shape-spiral@1.7.0
 - cra-template-particles-typescript@1.5.0
 - cra-template-particles@1.5.0
 - tsparticles-website@1.22.0
